styles.css

```diff
@@ -1,66 +1,67 @@
 html {
-    font-size: 10px;
+  font-size: 11px;
 }
 body {
```

Comment:
Kindly target the whole document and always give them a padding and margin of 0. You can also set box-sizing to border box😉. this usually ensures that elements you define in your document does not take up extra paddings or margins and they stay just as they have been defined 😄. To do this: 
*{
   margin: 0px; 
   padding: 0px; 
   box-sizing: border-box;
}